common/types.go

```diff
@@ -295,3 +297,5 @@ type WebhookVerificationParameters struct {
 	ClientSecret string
 	Method       string
 }
+
+type SubscriptionEventMap GenericMap
```

Comment:
SubscriptionEventMap isn't used anywhere, and you already have SubscriptionEvent in the `subscriptionEvent.go` file, this should be removed.